src/base/Multicall.sol

```diff
@@ -20,7 +26,7 @@ abstract contract Multicall is IMulticall {
                     }
                 }
                 // Next 5 lines from https://ethereum.stackexchange.com/a/83577
-                if (result.length < 68) revert();
+                if (result.length < 68) CallFailed.selector.bubbleUpAndRevertWith();
```

Comment:
not sure about this 68 length check.. but why wouldnt we want to bubble up if > 68 ?

Reply:
because in old-solidity errors couldnt be less than 68 bytes, so it meant there was no error to bubble